assets/src/scripts/wdfn-home/site-summary/index.js

```diff
@@ -0,0 +1,36 @@
+import { select } from 'd3-selection';
+import { link } from '../../lib/d3-redux';
+import { Sites } from '../selectors/wdfn-selector';
+
+const SiteSummaries = (node, store) => {
+    const siteSummaryTmpl = (siteName, siteId) => {
+      return `<li>
+                <h3>${siteName}</h3>
+                <p class="site-id">${siteId}</p>
+              </li>`;
+    };
+
+    const populateSiteSummaries = (node, sites) => {
+      const mapContainer = document.getElementById('wdfn-map-container');
+      const siteSummariesList = document.querySelector('#site-summaries ul');
+
+      if (sites && sites.length > 0) {
+        mapContainer.dataset.summariesVisible = 'true';
+
+        const siteListMarkup = sites.map(s => {
+          return siteSummaryTmpl(s.properties.MonitoringLocationName, 
+            s.properties.MonitoringLocationIdentifier);
+        });
+
+        siteSummariesList.innerHTML = siteListMarkup.join('');
```

Comment:
Stringifying an array implicitly calls `Array.prototype.join(',')` so explicitly call it here to join with an empty string to avoid rendering extraneous commas.